Makefile

```diff
@@ -1,10 +1,29 @@
+.PHONY: bake help quality replay requirements test upgrade validate watch
```

Comment:
Most of these don't match targets in the file; I'm guessing you grabbed them from the cookiecutter's `Makefile` instead of the one in the app template?